### PR TITLE
Update upgrade guide for v3.3

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/upgrade.md [hunk 1]
```diff
@@ -134,18 +134,20 @@ For any issues, Mattermost Enterprise Edition subscribers and trial license user
 
 ### Upgrade Enterprise Edition 
 
-#### Upgrade Enterprise Edition to 3.2.x
+#### Upgrade Enterprise Edition to 3.2.x and later
+
+Upgrading to `v3.2.x` and `v3.3.x` follows the procedure to [upgrade to `v3.1.x` and later](http://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x), but there is a breaking change where under `LdapSettings` in `config.json` `"FirstNameAttribute"` and `"LastNameAttribute"` are now required fields.
```
Comment (lindy65) [inline source/administration/upgrade.md]: Is `http://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x` missing the `s` from `https` ??

**Comment by jasonblais** [comment]
Thanks Lindy!

**Comment by lindy65** [comment]
Pleasure Jason! 

+1

**Comment by esethna** [comment]
+1, @jasonblais just need to verify the updated links are working when we do a docs correction pass after this is on master.

File: source/administration/upgrade.md [hunk 2]
```diff
@@ -134,20 +134,22 @@ For any issues, Mattermost Enterprise Edition subscribers and trial license user
 
 ### Upgrade Enterprise Edition 
 
-#### Upgrade Enterprise Edition to 3.2.x
+#### Upgrade Enterprise Edition to 3.2.x and later
+
+Upgrading to `v3.2.x` and `v3.3.x` follows the procedure to [upgrade to `v3.1.x` and later](https://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x), but there is a breaking change where under `LdapSettings` in `config.json` `"FirstNameAttribute"` and `"LastNameAttribute"` are now required fields.
```
Comment (enahum) [inline source/administration/upgrade.md]: can we make `are now required fields` in bold?

File: source/administration/upgrade.md [hunk 3]
```diff
@@ -134,20 +134,22 @@ For any issues, Mattermost Enterprise Edition subscribers and trial license user
...
 
-Upgrading to `v3.2.x` follows the procedure to [upgrade to `v3.1.x` and above](http://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition-to-3-1-x-and-later), but there is a breaking change where under `LdapSettings` in `config.json` `"FirstNameAttribute"`, `"LastNameAttribute"`, `"BindUsername"`, and `"BindPassword"` are now required fields. Anonymous bind is not currently supported. Other config changes can be reviewed in the [compatibility section of the changelog](http://docs.mattermost.com/administration/changelog.html?highlight=upgrade%20guide#config-json).
+Furthermore, for `v3.2.x` only, `"BindUsername"`, and `"BindPassword"` are also required fields with anonymous bind not supported.
```
Comment (enahum) [inline source/administration/upgrade.md]: same here

**Comment by enahum** [comment]
Other than my comments and a rebase +1

**Comment by jasonblais** [comment]
Thanks @enahum! Rebased and added bolding to the required fields

**Pull request merged**